Wikipedia/Code/WMFExploreCollectionViewController.m

```diff
@@ -1080,9 +1080,6 @@ - (nonnull UICollectionReusableView *)collectionView:(UICollectionView *)collect
             return [collectionView dequeueReusableSupplementaryViewOfKind:UICollectionElementKindSectionFooter withReuseIdentifier:WMFFeedEmptyHeaderFooterReuseIdentifier forIndexPath:indexPath];
         case WMFFeedMoreTypeLocationAuthorization: {
             WMFTitledExploreSectionFooter *footer = (id)[collectionView dequeueReusableSupplementaryViewOfKind:UICollectionElementKindSectionFooter withReuseIdentifier:[WMFTitledExploreSectionFooter wmf_nibName] forIndexPath:indexPath];
-            footer.titleLabel.text = [EnableLocationViewController localizedEnableLocationTitle];
-            footer.descriptionLabel.text = [EnableLocationViewController localizedEnableLocationDescription];
-            [footer.enableLocationButton setTitle:[EnableLocationViewController localizedEnableLocationButtonTitle] forState: UIControlStateNormal];
```

Comment:
My bad! I removed it cause It is overriding the "Explore articles near your current location" label to be "Explore articles near your location by enabling Location Access". I can add a new WMFLocalizedString with "Explore articles near your current location" and set `footer.titleLabel.text` to that (and obviously revert the rest). Does that sound like a good solution or can you think of a better one?